src/utils/appData.ts

```diff
@@ -1,4 +1,9 @@
 import Ajv, { ErrorObject, ValidateFunction } from 'ajv'
+import CID from 'cids'
+import fetch from 'cross-fetch'
+import multihashes from 'multihashes'
```

Comment:
Not sure how big are these libs, but ideally we would load them lazily. Not sure were is the best place to do the lazy loading and code splitting. 

Ideally we would avoid consuming apps that don't use this functions, to load the library.